nservicebus/azure-servicebus/topologies/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Azure Service Bus Topologies
+summary: Azure Service Bus Topologies
+tags:
+- Azure
+- Cloud
+redirects:
+related:
+ - samples/azure/azure-service-bus
+reviewed: 2016-04-12
+---
+
+Messaging topology is an arrangement of the various messaging entities.
+Azure Service Bus transport operates on a topology created on the broker. Messaging entities participating in topology are queues, topics, subscriptions, and rules. Combined in a specific way they form a topology. 
+
+Topology handles message exchange between endpoints by creating and configuring Azure Service Bus entities.  
+
+## Version 6 and below
+
+Version 6 and below of the transport has always supported a single default topology out-of-the-box. It was an implicit implementation of what was defined as `EndpointOrientedTopology` in version 7 and up.
```

Comment:
> Version 6 and below of the transport has always supported a single default topology out-of-the-box. It was an implicit implementation of what was defined as `EndpointOrientedTopology` in version 7 and up.

The Azure Service Bus transport has always supported a single default topology out-of-the-box. It is equivalent to the `EndpointOrientedTopology` introduced in Version 7 and higher.